docs/spire-multi-cluster.md

```diff
@@ -55,6 +56,23 @@ Procedures for obtaining these values are [here](https://docs.aws.amazon.com/gen
 For more information about this plugin configuration see the
 [attestor documentation](https://github.com/spiffe/spire/blob/main/doc/plugin_server_nodeattestor_aws_iid.md)
 
+* SPIRE Agents:
+To use AWS node_attestor, we need to provide the following value in
+[spire helm chart configuration file](../charts/spire/values.yaml):
+```yaml
+aws: true
+```
+
+Or add the `--set "aws=true"` flag to the helm command:
+```console
+helm install --set "spireServer.address=$SPIRE_SERVER" \
+--set "spireServer.port=$SPIRE_PORT"  --set "namespace=$AGENT_NS" \
+--set "clustername=$CLUSTERNAME" --set "region=us-east" \
+--set "trustdomain=openshift.space-x.com" \
+--set "aws=true" \
+spire charts/spire --debug
+```
```

Comment:
probably in the future, we'd need a subconfiguration for custom metadata endpoint, but should be ok for most deployments, and can support it if its asked for

Reply:
Captured in issue #116